Class-9-(Exceptions-&-Build-Tools).md

```diff
@@ -0,0 +1,33 @@
+# Class 9: Exceptions and Build Tools
+
+## In the prep work this week, the students learned
+
+1. What an exception is and how to handle an exception.
```

Comment:
Actually, we assume that students are familiar with the concept of an exception - the material is more intended to be a review of the idea and how to use them specifically in the c# language

Reply:
Rephrased!